### PR TITLE
more Less

### PR DESCRIPTION
**Pull request by radcliff**
Could use a lot more work on the CSS, currently only designed for < 320px

### CODE REVIEW

**Comment by stujo** [comment]
Hi Sterling, I'll be doing your code review on this project! Is it up to date with your latest and greatest?

**Comment by radcliff** [comment]
Yes. This commit represents completion of a design for mobile, which is where I stopped working on the project. When we go over it, we can check it out in the iOS simulator, or smaller browser window :) All of the functionality should be working.

File: movies.rb [hunk 1]
```diff
@@ -11,6 +12,22 @@
 end
 
 get '/' do
+	erb :index
 end
 
+get '/results' do
+	if params[:movie] == ""
+		redirect '/'
```
Comment (stujo) [inline movies.rb:20]: nice guard clause and use of params hash in API call

File: spec/movies_spec.rb [hunk 2]
```diff
@@ -9,5 +9,32 @@
     end
   end
 
+  describe "Results Page" do
```
Comment (stujo) [inline spec/movies_spec.rb:12]: Well done adding good tests

File: views/layout.erb [hunk 3]
```diff
@@ -1,9 +1,26 @@
-<!doctype html>
+<!-- template -->
+<!DOCTYPE html>
 <html>
 <head>
-  <title>Your Title Here</title>
+	<meta name="viewport" content="width=device-width, user-scalable=no">
+	<link rel="stylesheet/less" type="text/css" href="/stylesheets/less/bootstrap.less">
```
Comment (stujo) [inline views/layout.erb:6]: Nice use of less, we'll probably look into how to do this for a production release when we have gone over the rails asset pipeline

**Pull request closed without merging**